src/main/java/net/uaznia/lukanus/hudson/plugins/gitparameter/GitParameterDefinition.java

```diff
@@ -290,10 +300,12 @@ public String prettyRevisionInfo(GitClient newgit, Revision r) {
 					return Collections.singletonMap(errMsg, errMsg);
 				}
 
-				FetchCommand fetch = newgit.fetch_().from(remoteURL,
-						repository.getFetchRefSpecs());
-				fetch.execute();
 				if (type.equalsIgnoreCase(PARAMETER_TYPE_REVISION)) {
+					// If we use the GitClient#getRemoteReferences function, we don't
```

Comment:
The ls-remote commit was reverted and moved into the lsremote branch: https://github.com/jenkinsci/git-parameter-plugin/commits/lsremote

However, after doing that, I now see that @ndeloof had already created the [ls-remote](https://github.com/jenkinsci/git-parameter-plugin/commits/ls-remote) branch to do the same thing.

The main difference is that my `lsremote` branch makes "prefer remote" an option that can be enabled.  When enabled, it will use `ls-remote` to get tags and/or branches, otherwise it falls back on existing behavior requiring a workspace.  Using the "revision" type also uses the workspace.

I'll defer to @ndeloof to determine how to proceed forward, whether to revive his change or use my optional version.